1-JavaScript/Week4/homework/ex2-mondaysWorth.js

```diff
@@ -55,7 +64,7 @@ const earnings = computeEarnings(
   ],
   hourlyRate
 );
-console.log(`Total earnings: ${earnings}`); // -> "Total earnings: €212.5"
+console.log(`Total earnings: ${earnings1}`); // -> "Total earnings: €212.5"
```

Comment:
Your implementation is correct, with one sidenote. If you run the test for this exercise, you'll see that it finds 1 thing:

```
- mondaysWorth should compute the earnings as a formatted Euro amount
  Expected: €187.50
  Received: €187.5
```

(So actually the comment is a bit wrong here) We've looked at how to do this during one of the Q&A's, think you can make the code add the zero?